packages/@lwc/ssr-compiler/src/__tests__/utils/expected-failures.ts

```diff
@@ -7,43 +7,107 @@
+        // missing slot attr, bonus bookends
+        'slot-forwarding/slots/dangling/index.js',
+        // bonus bookends
+        'slot-forwarding/slots/light/index.js',
+    ],
+    'TO BE FILED: align error messages': [
+        // divergent error
+        'wire/errors/throws-on-computed-key/index.js',
+        // divergent error
+        'wire/errors/throws-when-colliding-prop-then-method/index.js',
+        // divergent error
+        'wire/errors/throws-when-computed-prop-is-expression/index.js',
+        // divergent error
+        'wire/errors/throws-when-computed-prop-is-let-variable/index.js',
+        // divergent error
+        'wire/errors/throws-when-computed-prop-is-regexp-literal/index.js',
+        // divergent error
+        'wire/errors/throws-when-computed-prop-is-template-literal/index.js',
+        // divergent error
+        'wire/errors/throws-when-using-2-wired-decorators/index.js',
+    ],
+    'TO BE FILED: add validation': [
+        // renders but shouldn't
+        'wire/errors/throws-when-wired-method-is-combined-with-@api/index.js',
+        // renders but shouldn't
+        'wire/errors/throws-when-wired-property-is-combined-with-@api/index.js',
+        // renders but shouldn't
+        'wire/errors/throws-when-wired-property-is-combined-with-@track/index.js',
+    ],
+    'TO BE FILED: fix superclass rendering': [
+        // missing content
+        'superclass/render-in-superclass/no-template-in-subclass/index.js',
+        // incorrect content
+        'superclass/render-in-superclass/unused-default-in-subclass/index.js',
+        // missing content
+        'superclass/render-in-superclass/unused-default-in-superclass/index.js',
+    ],
+    'TO BE FILED: ARIA': [
+        // aria attributes not rendered
+        'attribute-aria/dynamic/index.js',
+    ],
+    'TO BE FILED: global attributes': [
+        // some props not rendered
+        'attribute-component-global-html/index.js',
+        // incorrect values rendered
+        'attribute-global-html/as-component-prop/undeclared/index.js',
+        // incorrect draggable rendered
+        'known-boolean-attributes/default-def-html-attributes/static-on-component/index.js',
+    ],
+    'TO BE FILED: style': [
+        // incorrect style rendered
+        'attribute-style/basic/index.js',
+        // incorrect style rendered
+        'attribute-style/dynamic/index.js',
+    ],
+    'TO BE FILED: scope token': [
+        // scope token not rendered
+        'attribute-class/with-scoped-styles-only-in-child/dynamic/index.js',
+        // scope token not rendered
+        'attribute-class/with-scoped-styles/dynamic/index.js',
+    ],
+    'TO BE FILED: unique issues': [
+        // aria attributes not rendered
+        'attribute-global-html/as-component-prop/without-@api/index.js',
+        // test fixture formatting needs fixed?
+        'render-dynamic-value/index.js',
+        // throws error
+        'exports/component-as-default/index.js',
+    ],
+};
```

Comment:
Why not keep it as a flat list but then use code comments for the WIs?

Reply:
Because code comments aren't going to _force_ us to keep the work items up to date. It's a token amount of nuisance so that we do the thing we're supposed to.

Reply:
I like the idea, but as an external contributor I would appreciate if github issues can be used instead, and have those linked to gus.

Another thing to consider is that tickets and expected failures seem to have a many-to-many relationship rather than one-to-many. But I guess most if not all have a main ticket, so this structure might be fine.

Reply:
I don't have a super strong opinion on this, since eventually we will drive this list to zero. But yes including a GitHub issue along with a WI is a good idea. Most of these WIs should have an associated GitHub issue.